.github/workflows/test.yml

```diff
@@ -48,13 +48,28 @@ jobs:
 
   test-submodules:
```

Comment:
can we use this docker image instead 

```
 container: tendermintdev/docker-tm-db-testing
```

we are using it to test rocksdb elsewhere already

Reply:
Okay, this has Rocksdb 6.6.4 though, we will need to update it

Reply:
PR for update https://github.com/tendermint/images/pull/75

Reply:
sorry, this was moved here: https://github.com/tendermint/tm-db/blob/master/tools/Dockerfile. Can you move your pr here and ill merge right away

Reply:
New PR - https://github.com/tendermint/tm-db/pull/197

Reply:
this should be good to go now